packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -70,7 +70,8 @@ export class ArtifactCreator {
       mkdirp.sync(scenarioOutputDirectory);
 
       const screenshot = await this.captureScreenshot(
-          'model-viewer',
+          'model-viewer',  // hard code to model viewer?  how do you load other
+                           // renders?
```

Comment:
This is for the captureAndAnalyze function, which we currently only run on model-viewer (that's the only one we generate statistics for). As we add other renderers, and especially when we have better ground truth renders to compare to, we should revisit how these scripts should operate.